.github/stale.yml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
```

Comment:
The configuration labels are enough explicit, I think the comments can be removed from this file.

Reply:
It's ok. The comments were removed.